test/suites/edgex-no-sec/status_test.go

```diff
@@ -20,8 +22,17 @@ func TestServiceStatus(t *testing.T) {
 
 func TestAccess(t *testing.T) {
 	t.Run("consul", func(t *testing.T) {
+		t.Log("Getting Consul token")
+		consulToken, stderr, err := utils.Exec(t, fmt.Sprintf("sudo cat /var/snap/edgexfoundry/current/secrets/consul-acl-token/bootstrap_token.json | jq -r '.SecretID'"))
+		require.Empty(t, stderr)
+		require.NoError(t, err)
```

Comment:
1. Extra formatting.
2. utils.Exec takes care of error handling internally, when `*testing.T` is passed to it.
```suggestion
		consulToken, _, _ := utils.Exec(t, "sudo cat /var/snap/edgexfoundry/current/secrets/consul-acl-token/bootstrap_token.json | jq -r '.SecretID'")
```